lib/mongo/collection/view/writable.rb

```diff
@@ -244,6 +253,19 @@ def update(spec, multi, opts)
             ).execute(server)
           end
         end
+
+        def apply_array_filters!(doc, server, opts = {})
+          if filters = opts[:array_filters] || opts[:arrayFilters] || opts['array_filters'] || opts['arrayFilters']
```

Comment:
I put the constant at the top of this module; let me know if there's somewhere else that would be better suited for it.